### PR TITLE
chore: remove all ants from the toolbar

### PR DESCRIPTION
**Pull request by pauldambra**
The toolbar css is 85.9kb compressed 716kb uncompressed

I wonder if that gets better without antd?

## Changes

Adds a couple of stories, and removes all toolbar ants.
Doesn't cover all of the interactions with stories.
Locally this reduces the CSS from 800+kb to 234kb

| before | after |
|-|-|
| Screenshot 2023-08-27 at 15 23 22|Screenshot 2023-08-27 at 15 39 34|
|Screenshot 2023-08-27 at 15 22 05|Screenshot 2023-08-27 at 15 36 19|

## How did you test this code?

storybook and 👀 locally

### CODE REVIEW

File: frontend/src/toolbar/actions/ActionsList.tsx [hunk 1]
```diff
@@ -13,22 +14,21 @@ export function ActionsList(): JSX.Element {
...
+                onChange={(s) => setSearchTerm(s)}
             />
             <div className="actions-list">
-                <Row className="actions-list-header">
-                    <Button type="primary" size="small" onClick={() => newAction()} style={{ float: 'right' }}>
-                        <PlusOutlined /> New action
-                    </Button>
-                </Row>
+                <div className="actions-list-header">
+                    <LemonButton type="primary" size="small" onClick={() => newAction()} icon={<IconPlus />}>
```
Comment (pauldambra) [inline frontend/src/toolbar/actions/ActionsList.tsx:28]: this float right didn't apply so the button has always been on the left 🤷

File: frontend/src/toolbar/actions/ActionsTab.scss [hunk 2]
```diff
@@ -1,9 +1,17 @@
 .action-block-body {
-    padding: 16px;
+    form {
+        .form-error {
+            color: var(--danger);
+        }
+
+        //// TODO why doesn't this match?
```
Comment (daibhin) [inline frontend/src/toolbar/actions/ActionsTab.scss:7]: Can we remove this entirely?

**Review by daibhin — APPROVED** [review]
Amazing!

File: frontend/src/toolbar/actions/ActionsTab.scss (hunk 2 above)
Comment (pauldambra) [reply frontend/src/toolbar/actions/ActionsTab.scss:7]: me: "I'll definitely remember to check for TODOs before I'm finished"
:side-eye-monkey-dot-meme:

File: frontend/src/toolbar/actions/ActionsTab.scss [hunk 3]
```diff
@@ -1,9 +1,17 @@
...
+        //.LemonInput__input[error] {
+        //    border-color: var(--danger);
+        //}
```
Comment (pauldambra) [inline frontend/src/toolbar/actions/ActionsTab.scss:10]:
```suggestion
```

**Pull request merged**